vraptor-core/src/main/java/br/com/caelum/vraptor/VRaptor.java

```diff
@@ -52,7 +52,7 @@
  * @author Fabio Kung
  */
 public class VRaptor implements Filter {
-	private static final String VERSION = "3.5.4-SNAPSHOT";
+	private static final String VERSION = "3.5.5-SNAPSHOT";
```

Comment:
@MarceloPortilho could you please rollback this change?
This version will automatically be updated when we release `3.5.5`.